plant.rb

```diff
@@ -0,0 +1,24 @@
+class Plant
+
```

Comment:
This take the full API response and then filters the data we want to use in the app.